pyworkflow/apps/pw_run_tests.py

```diff
@@ -206,7 +206,7 @@ def _visitTests(self, moduleName, tests, newItemCallback):
     def _printNewItem(self, itemType, itemName):
         if self._match(itemName):
             spaces = (itemType * 2) * ' '
-            print("%s %s %s" % (spaces, self.getTestsCommand(), itemName))
+            print("%s scipion test %s" % (spaces, itemName))
```

Comment:
This will clash with current launcher that is scipion3. Maybe we can default to this value inside getTestsCommand() ?

Reply:
getTestsCommand() prints the whole script path. 

I'm still using 'scipion' as the main launcher...should we define an environment variable for that?

Reply:
Either  'scipion' or 'scipion3', I  think the test command should be more legible and easier to copy/paste to run individual tests.

Reply:
https://github.com/scipion-em/scipion-pyworkflow/blob/devel/pyworkflow/apps/pw_run_tests.py#L212

It is already there: SCIPION_TESTS_CMD

Reply:
Right! Great...although it should be better in pw.Config I think

Reply:
I reverted this change.